### DISCUSSION TITLE
Unknown interaction

### DISCUSSION THREAD

**Discussion by everlyy** [Rapptz/discord.py#8418] [Q&A] [answered]
I keep getting this error:
`discord.errors.NotFound: 404 Not Found (error code: 10062): Unknown interaction`
and after some testing, I figured out it's because I'm taking too long to respond to the interaction, because I'm making a few web requests etc. is there a way to extent my period of time to respond to the interaction or something else?

**Comment by Bluesy1** [comment, marked as answer]
You can use the following to defer your response:

```py
await interaction.response.defer()
# later
await interaction.followup.send(...)
```
[Docs here](https://discordpy.readthedocs.io/en/latest/interactions/api.html#discord.InteractionResponse.defer)
Note: the epehemerality of the response is set in `defer()`
